src/components/feed/actions/goes-to-page.tsx

```diff
@@ -0,0 +1,31 @@
+// Modules
+import Link from "next/link";
+
+// Types
+import { ActionOpensLink } from "@utils/types/feed/action";
+
+/**
+ * This action goes to a page within Colony School
+ * @param action An action object
+ */
+const ActionGoesToPage = ({ action }: { action: ActionOpensLink }) => {
+  return (
+    <Link href={action.url}>
+      <a
+        key={action.name}
+        className={`btn ${
+          action.importance == 1
+            ? "btn-filled"
+            : action.importance == 2
+            ? "btn-outlined"
+            : action.importance == 3 && "btn-text"
```

Comment:
What if someone inputs 4?
You could just remove the `action.importance == 3` and just return the `btn-text` as the fallback